src/main/java/mekanism/common/integration/multipart/MultipartTransmitter.java

```diff
@@ -28,6 +30,28 @@ public IPartSlot getSlotFromWorld(IBlockAccess world, BlockPos pos, IBlockState
 	}
 	
 	@Override
+	public void onAdded(IPartInfo part)
+	{
+		TileEntity tile = part.getTile().getTileEntity();
+		
+		if(tile instanceof TileEntitySidedPipe) 
+		{
+			((TileEntitySidedPipe)tile).onPartChanged(null);
+		}
+	}
+	
+	@Override
+	public void onPartAdded(IPartInfo part, IPartInfo otherPart)
+	{
+		TileEntity tile = part.getTile().getTileEntity();
+		
+		if(tile instanceof TileEntitySidedPipe) 
+		{
+			tile.validate();
```

Comment:
Unfortunately it does because the override of `invalidate` calls `onWorldSeparate`